uptrain/operators/language/sql.py

```diff
@@ -0,0 +1,185 @@
+"""
+Implement oeprators over sql data.
+"""
+
+from __future__ import annotations
+
+import itertools
+import json
+import os
+import typing as t
+
+import sqlglot
```

Comment:
Can you make the dependency on sqlglot as lazy load? Check: https://github.com/uptrain-ai/uptrain/blob/8d71e84d73dedf22948d4dbd7da77fb67b26d38d/uptrain/operators/embs.py#L18C1-L19C1